iOS/DuckDuckGo/DownloadsDirectoryHandling.swift

```diff
@@ -0,0 +1,67 @@
+//
```

Comment:
I’ve extracted directory handling functions from `DownloadManager` into this new struct so it can be reused.